transform_data.py

```diff
@@ -49,15 +59,37 @@ def standardize_price_to_usd(stock_history:pd.DataFrame) -> pd.DataFrame:
         pd.DataFrame: stock history with close price convert in USD
     """
     try:
-        stock_ticker = stock_history['stock'].iloc[0]
-        currency_code = ed.get_stock_currency_code(stock_ticker)
-        if currency_code == 'USD':
-            stock_history['usd_close'] = stock_history['Close']
+        stock_ticker = stock_history["stock"].iloc[0]
+        currency_code = get_stock_currency_code(stock_ticker)
+        if currency_code == "USD":
+            stock_history["usd_close"] = stock_history["Close"]
```

Comment:
create a model.py and catogerize columns into one class object

from enum import Enum

StockPrice(Enum):
     CLOSE = 'Adj Close'
     TICKER = 'ticker'



FXRate(Enum):
    FROM_CURRENCY = 'from curreny"
    TO_CURRENCY

 stock_history["usd_close"] = stock_history[StockPrice.CLOSE.value]